app/presenters/menu/default_menu.rb

```diff
@@ -2,6 +2,13 @@
 module Menu
   class DefaultMenu
     class << self
+      def graphql_menu_section
+        return nil unless Rails.env.development?
+        Menu::Section.new(:graphql, N_("GraphQL"), 'fa fa-lightbulb-o', [
+          Menu::Item.new('graphql', N_('Explorer'), 'utilization', {:feature => 'utilization'}, '/graphql_explorer'),
```

Comment:
We will need a new feature id here. `utilization` is already being used for something else.

Reply:
Although, if this is just meant to be used in the development environment, then it does not really matter if we have a feature id or not.

```rb
Menu::Item.new('graphql', N_('Explorer'), '', {:feature => ''}, '/graphql_explorer'),
```
should also work.